### DISCUSSION TITLE
filter for the linkToAnotherRecord field [Apr 29]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#1987] [Discord Threads]
> 🙋‍♂️ **vishal.gandhi**  
## How do I filter for the `linkToAnotherRecord` field?  
![image0](https://cdn.discordapp.com/attachments/661905455894888493/968916912610373714/unknown.png)

**Comment by o1lab** [comment]
> 👨‍🚀 **wingkwong**  
#### filter for LinkToAnotherRecord s supported for version 0.90.x. it seems u r still using 0.84.x  
#  
> 🙋‍♂️ **vishal.gandhi**  
#### Thank you. How do i check the version ? I am running latest docker image  
#  
> 🙋‍♂️ **vishal.gandhi**  
#### 140d2fbf7d5   nocodb/nocodb:latest  
#  
> 🙋‍♂️ **vishal.gandhi**  
#### repulled the image ; able to see filter now  
#  
> 🙋‍♂️ **vishal.gandhi**  
#### thank you  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/968916913294016652)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
